src/pages/Loans/components/LoanCards/CloseLoanCard/CloseLoanCard.js

```diff
@@ -68,13 +67,9 @@ export const CloseLoanCard = ({
 		try {
 			const loanIDStr = loanID.toString();
 
-			const gasEstimate = await contract.estimate.closeLoan(loanIDStr);
```

Comment:
I agree that we should have a custom gasLimit when we actually send the tx, however, won't the `closeLoan` catch failing tx?

Just thinking if we should just
```
await contract.estimate.closeLoan(loanIDStr);
```

without actually doing anything with the value